google-ads-migration-examples/src/main/java/com.google.ads.googleads.migration/createcampaign/CreateCompleteCampaignAdWordsApiOnly.java

```diff
@@ -0,0 +1,429 @@
+    networkSetting.setTargetContentNetwork(false);
+    networkSetting.setTargetPartnerSearchNetwork(false);
+    campaign.setNetworkSetting(networkSetting);
+
+    // Create the operation.
+    CampaignOperation operation = new CampaignOperation();
+    operation.setOperand(campaign);
+    operation.setOperator(Operator.ADD);
+
+    CampaignOperation[] operations = new CampaignOperation[] {operation};
+
+    // Add the campaign.
+    CampaignReturnValue result = campaignService.mutate(operations);
+
+    Campaign campaignResult = result.getValue(0);
+    // Display the campaign.
+    System.out.printf("Campaign with ID '%d' and name %s was created.%n",
+      campaignResult.getId(),
+      campaignResult.getName());
+    return campaignResult;
+  }
+
+  /**
+   * Creates a budget.
+   *
+   * @param adWordsServices the Google AdWords services interface.
+   * @param session the client session.
+   * @throws RemoteException if the API request failed due to other errors.
+   */
+  private Budget createBudget(AdWordsServicesInterface adWordsServices,
+                                      AdWordsSession session) throws RemoteException {
+    // Get the BudgetService.
+    BudgetServiceInterface budgetService =
+      adWordsServices.get(session, BudgetServiceInterface.class);
+
+    // Create a budget, which can be shared by multiple campaigns.
+    Budget sharedBudget = new Budget();
+    sharedBudget.setName("Interplanetary Cruise #" + System.currentTimeMillis());
+    Money budgetAmount = new Money();
+    budgetAmount.setMicroAmount(500_000L);
+    sharedBudget.setAmount(budgetAmount);
+    sharedBudget.setDeliveryMethod(BudgetBudgetDeliveryMethod.STANDARD);
+
+    BudgetOperation budgetOperation = new BudgetOperation();
+    budgetOperation.setOperand(sharedBudget);
+    budgetOperation.setOperator(Operator.ADD);
+
+    BudgetOperation[] operations = new BudgetOperation[] {budgetOperation};
+
+    // Add the budget.
+    BudgetReturnValue result = budgetService.mutate(operations);
+    Budget budgetResult = result.getValue(0);
+    // Display budget.
+    System.out.printf("Budget with ID %d and name %s was created.%n",
+      budgetResult.getBudgetId(),
+      budgetResult.getName());
+    return budgetResult;
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param adWordsServices the Google AdWords services interface.
+   * @param session the client session.
+   * @throws RemoteException if the API request failed due to other errors.
+   * @throws UnsupportedEncodingException if encoding the final URL failed.
+   */
+  private void runExample(AdWordsServicesInterface adWordsServices, AdWordsSession session)
+    throws RemoteException, UnsupportedEncodingException {
+    Budget budget = createBudget(adWordsServices, session);
+    Campaign campaign = createCampaign(adWordsServices, session, budget);
+    AdGroup adGroup = createAdGroup(adWordsServices, session, campaign);
+    createTextAds(adWordsServices, session, adGroup);
```

Comment:
For consistency, please pass NUMBER_OF_ADS to `createTextAds`. That'll also help with readability of that method.

Reply:
Done.